spec/acceptance/users/view_user.feature

```diff
@@ -4,7 +4,8 @@ Feature: User Viewing
   So I can see users data on user's page
 
   Scenario: user can see users data on user page
-    Given I logged to the system as user
+    Given registered user in the system
```

Comment:
The main rule:
1 - all prerequisites via factory should be created first, then UI steps

Example:
Given there are test users in the system
And I am logged in as admin
And I am on users list page

Simple and clear for reading